main/glossary/README.md

```diff
@@ -354,6 +350,18 @@ the user either gets what they said they wanted, or gets back (gets a refund) wh
 escrowed. One reason this is possible is if a [proposal](#proposal) doesn't match what the contract expects to do, it
 can immediately cause the [seat](#seat) to exit, getting back the amount it offered.
 
+
+## Passable
+
+A *passable* is something that can be marshalled.
+
+There are three kinds of passables:
+   * Remotables, objects with methods that can be called remotely using `E()`, and their remote Presence.
+   * Pass-by-copy data, such as numbers or hardened records.
+   * Promises for passables.
```

Comment:
This is a bit circular. How about "Promises for either remotables or pass-by-copy data." to be clearer?

Reply:
Saying "pass-by-copy data" would exclude remotables. But promises can be fulfilled to remotables, so that would not be correct. 

The truth is exactly as circular as it sounds. A promise can be fulfilled to any passable. To be fully accurate, a promise can be fulfilled to any non-promise passable.